GameDataParser/Parsers/AdditionalEffectParser.cs

```diff
@@ -49,82 +49,231 @@ protected override List<AdditionalEffectMetadata> Parse()
-                    AddStat(level.Status, StatAttribute.CritRate, stat.capvalue);
-                    AddStat(level.Status, StatAttribute.CritDamage, stat.cadrate);
-                    AddStat(level.Status, StatAttribute.CritDamage, stat.cadvalue);
-                    AddStat(level.Status, StatAttribute.CritEvasion, stat.carrate);
-                    AddStat(level.Status, StatAttribute.CritEvasion, stat.carvalue);
-                    AddStat(level.Status, StatAttribute.BonusAtk, stat.baprate);
-                    AddStat(level.Status, StatAttribute.BonusAtk, stat.bapvalue);
-                    AddStat(level.Status, StatAttribute.MountMovementSpeed, stat.rmsprate);
-                    AddStat(level.Status, StatAttribute.MountMovementSpeed, stat.rmspvalue);
-                    AddStat(level.Status, StatAttribute.Pierce, stat.penrate);
-                    AddStat(level.Status, StatAttribute.Pierce, stat.penvalue);
-                    AddStat(level.Status, StatAttribute.Damage, stat.dmgrate);
-                    AddStat(level.Status, StatAttribute.Damage, stat.dmgvalue);
-                    AddStat(level.Status, StatAttribute.MinWeaponAtk, stat.wapminrate);
-                    AddStat(level.Status, StatAttribute.MinWeaponAtk, stat.wapminvalue);
-                    AddStat(level.Status, StatAttribute.MaxWeaponAtk, stat.wapmaxrate);
-                    AddStat(level.Status, StatAttribute.MaxWeaponAtk, stat.wapmaxvalue);
-                    AddStat(level.Status, StatAttribute.MagicPiercing, stat.marrate);  // revisit these to make sure they're right
-                    AddStat(level.Status, StatAttribute.MagicPiercing, stat.marvalue); // revisit these to make sure they're right
-                    AddStat(level.Status, StatAttribute.MagicAtk, stat.maprate);
-                    AddStat(level.Status, StatAttribute.MagicAtk, stat.mapvalue);
-                    AddStat(level.Status, StatAttribute.PhysicalPiercing, stat.parrate);  // revisit these to make sure they're right
-                    AddStat(level.Status, StatAttribute.PhysicalPiercing, stat.parvalue); // revisit these to make sure they're right
-                    AddStat(level.Status, StatAttribute.PhysicalAtk, stat.paprate);
-                    AddStat(level.Status, StatAttribute.PhysicalAtk, stat.papvalue);
-                    AddStat(level.Status, StatAttribute.JumpHeight, stat.jmprate);
-                    AddStat(level.Status, StatAttribute.JumpHeight, stat.jmpvalue);
-                    AddStat(level.Status, StatAttribute.PerfectGuard, stat.abprate);
-                    AddStat(level.Status, StatAttribute.PerfectGuard, stat.abpvalue);
-                    AddStat(level.Status, StatAttribute.PetBonusAtk, stat.bap_petrate);
-                    AddStat(level.Status, StatAttribute.PetBonusAtk, stat.bap_petvalue);
+                    AddStat(level.Status, StatAttribute.Str, stat.strvalue, stat.strrate);
```

Comment:
its not possible to do a for each with these stats?

Reply:
They're not quite in the same order, and value & rate happen separately. I don't know if I can trust a loop to do it, although I would love for a cleaner way to do this to exist. That was pain to write